lib/licensee/license.rb

```diff
@@ -70,6 +78,19 @@ def keys_licenses(options = {})
     # Note: A lack of detected license will be a nil license
     PSEUDO_LICENSES = %w[other no-license].freeze
 
+    # Default options to use when retrieving licenses via #all
+    DEFAULT_OPTIONS = {
+      hidden:   false,
+      featured: nil,
+      psuedo:   true
+    }.freeze
+
+    ALT_TITLE_REGEX = {
+      'bsd-2-clause'       => /bsd 2-clause(?: \"simplified\")?/i,
+      'bsd-3-clause'       => /bsd 3-clause(?: \"new\" or \"revised\")?/i,
+      'bsd-3-clause-clear' => /(?:clear bsd|bsd 3-clause(?: clear)?)/i
```

Comment:
Not related to the purpose of this PR (this regex definition is just moved), but just asking while noticing -- the purpose of this is to catch title matches that don't include bits that aren't needed to disambiguate, or also in the case of "clear bsd", an alternate name, right?

If so does it make sense to also add entries for the A/L/GPL licenses matching titles without the "GNU " prefix?

Reply:
>  or also in the case of "clear bsd", an alternate name, right?

Exactly. This allows `clear BSD` to match `bsd-3-clause-clear` since there is no ambiguity and some license templates had the alternate wording (for whatever reason).

> If so does it make sense to also add entries for the A/L/GPL licenses matching titles without the "GNU " prefix?

Yes. I can do that here very easily with this refactor. We can make `GNU` optional in all cases.

Reply:
> I can do that here very easily with this refactor. We can make GNU optional in all cases.

Implemented via 25e7388.